src/test/resources/logging.properties

```diff
@@ -30,6 +30,7 @@ logger.org.wildfly.security.level=TRACE
 logger.org.wildfly.security.sasl.level=TRACE
 logger.org.wildfly.security.xml.level=TRACE
 logger.org.wildfly.security.tls.level=TRACE
+logger.org.wildfly.security.html.level=TRACE
```

Comment:
I repeat: please remove this line - we have no html logger... :weary:

Reply:
I added html logger for HttpAuthenticator and HttpScope, so should I remove it and make them use log instead? It is also used by http.util classes, but if I understood correctly they will be merged with sasl util classes so I am supposed to change it back to log. Right?

Reply:
It is ok, but it is `org.wildfly.security.http`, not `html`.

Reply:
I must be really blind haha. I will change it back to log